GSoC-Ideas.md

```diff
@@ -38,4 +38,28 @@ The aims of the project are as follows:
 * _Mentors:_ Sean Goggins, Matt Germonprez
 
 
-## Idea: TBD
+## Idea: Implement the Social Currencyy Metrics System in Grimoire Labs
+
+[ Micro-tasks and place for questions ](https://github.com/chaoss/augur/issues/545)
+
+The Social Currency Metrics System (SCMS) is a qualitative data collection, processing, and measurement system that augments quantitative community management metrics already available in CHAOSS properties. Implementing the SCMS will ultimately help community leaders, power users, and other stakeholders leverage qualitative data for social listening so that they can rely less on the behaviors quantitative data tracks and more on community sentiment. The SCMS empowers community leaders to make decisions based on what community members freely share about their opinions, wants, and needs.
```

Comment:
Remove 'ultimately' in: 
Implementing the SCMS will ultimately help community leaders, power users, and other stakeholders leverage qualitative data for social listening so that they can rely less on the behaviors quantitative data tracks and more on community sentiment.